.env

```diff
@@ -1,4 +1,3 @@
-MONGO_USER = "codeuino-user"
-MONGO_PASSWORD = "thisisthewrongpassword"
+DATABASE_URL="mongodb://<username>:<password>@cluster0-shard-00-00.t4hho.mongodb.net:27017,cluster0-shard-00-01.t4hho.mongodb.net:27017,cluster0-shard-00-02.t4hho.mongodb.net:27017/<dbname>?ssl=true&replicaSet=atlas-a80apd-shard-0&authSource=admin&retryWrites=true&w=majority"
```

Comment:
You might want to remove this URL as it will be different for every mongo cluster.

Reply:
Yeah! Basically it was just a placeholder to make it clear on what type of URL is expected